command/certificate/verify.go

```diff
@@ -164,6 +177,30 @@ func verifyAction(ctx *cli.Context) error {
 		}
 	}
 
+	if expire {
+
+		NowTillEndOfCert := time.Until(cert.NotAfter)
+		totalLifeTimeOfCert := cert.NotAfter.Sub(cert.NotBefore)
+
+		percentIntoLifeTime := ((totalLifeTimeOfCert.Hours() - NowTillEndOfCert.Hours()) / totalLifeTimeOfCert.Hours()) * 100
+
+		if percentIntoLifeTime >= 100 {
+			fmt.Println("\033[31m", "This certificate has already expired.", "\033[0m") //"\033[__m" are color codes
```

Comment:
Just gonna list some bullet points here for improvements to this section: 

```
if percent >= 100 {
}
else if percent >= 90 {
}
else if percent >= 66 {
}
else if percent >= 1 {
}
else if precent >= 0 {
} else {
}
```
* Use `fmt.Printf` instead of `fmt.Println` if possible.
* parameterize commonly used strings and colors so you can reuse them. e.g `var colorExpired = "31m"`.